Cannot find module Source-map-support

Hi, I'm trying to push my build on production and I'm getting this error

`Cannot find module '/Users/NAME/FOLDER/PROJECT_NAME/node_modules/source-map-support/source-map-support.js'`

Anyone know how to fix this?

Thank you very much!

I have the same issue on Windows 7 64bit Enterprise
`yarn start`
```
$ node ./build/main.js
module.js:472
    throw err;
    ^

Error: Cannot find module 'C:codejsproductssfi.sebn.coackendattachment
ode_modulessource-map-supportsource-map-support.js'
    at Function.Module._resolveFilename (module.js:470:15)
    at Function.Module._load (module.js:418:25)
    at Module.require (module.js:498:17)
    at require (internal/module.js:20:19)
    at Object.<anonymous> (C:\code\js\products\sfi.sebn.com\backend\attachment\build\main.js:1:63)
    at Module._compile (module.js:571:32)
    at Object.Module._extensions..js (module.js:580:10)
    at Module.load (module.js:488:32)
    at tryModuleLoad (module.js:447:12)
    at Function.Module._load (module.js:439:3)
error Command failed with exit code 1.
```

```es6
console.log(require.resolve('source-map-support'))
// C:\code\js\products\sfi.sebn.com\backend\attachment\node_modules\source-map-support\source-map-support.js
console.log(`require('${require.resolve('source-map-support')}')`)
// require('C:\code\js\products\sfi.sebn.com\backend\attachment\node_modules\source-map-support\source-map-support.js')
```

package.json extract
```json
"scripts": {
  "start": "node ./build/main.js",
  "prestart": "backpack build"
}
```

versions
```
node: '7.5.0'
yarn: '0.19.1'
backpack-core: '0.0.9'
```

--- a/packages/backpack-core/config/webpack.config.js
+++ b/packages/backpack-core/config/webpack.config.js
@@ -115,7 +115,14 @@ module.exports = (options) => {
       new webpack.BannerPlugin({
         raw: true,
         entryOnly: false,
-        banner: `require('${require.resolve('source-map-support/register')}')`
+        banner: `require('${
+            // Is source-map-support installed as project dependency, or linked?
+            require.resolve('source-map-support').match(process.cwd())
+            // If it's resolvable from the project root, it's a project dependency.
+            ? 'source-map-support/register'
+            // It's not under the project, it's linked via lerna.
+            : require.resolve('source-map-support/register')
+        }')`
       }),
       // The FriendlyErrorsWebpackPlugin (when combined with source-maps)
       // gives Backpack its human-readable error messages.

